recipes/default.rb

```diff
@@ -21,7 +21,7 @@
 # Generate config file
 settings = {}
 node['uchiwa']['settings'].each do |k, v|
-  settings[k] = v
+  settings[k] = v unless v.nil?
```

Comment:
This makes it possible to not specify required config, I'd to a check on k and look for the specific ones to allow being nil and then check those for nil and warn on them.